save-backend/src/main/kotlin/com/saveourtool/save/backend/controllers/vulnerability/VulnerabilityController.kt

```diff
@@ -133,4 +133,20 @@ class VulnerabilityController(
     }.map {
         ResponseEntity.ok("Projects were successfully saved in vulnerability")
     }
+
+    @DeleteMapping("/delete-project")
+    @Operation(
+        method = "Delete",
+        summary = "Delete project.",
+        description = "Delete project.",
+    )
+    @ApiResponse(responseCode = "200", description = "Successfully deleted project in vulnerability")
+    fun deleteProject(
+        @RequestParam name: String,
```

Comment:
```suggestion
        @RequestParam projectName: String,
```

just in case of readability